src/firefly_exchange_client/api_service.py

```diff
@@ -60,9 +66,10 @@ async def post(self, service_url, data, auth_required=False, contentType=""):
         response = None
         if auth_required:
             headers = {
-                'Authorization': 'Bearer {}'.format(self.auth_token),
-                'x-mm-id': self.uuid or ''
+                'Authorization': 'Bearer {}'.format(self.auth_token)
             }
+            if self.uuid and self.uuid is not "":
+                headers['x-mm-id'] = self.uuid
```

Comment:
Just to confirm, this APIService is a wrapper and so this will ensure the header gets added to all post requests?

If so, shouldn't we also add the same to the `get()` and `delete()` routes as well?